src/Microsoft.Graph/Generated/Models/AgreementCollectionResponse.cs

```diff
@@ -1,13 +1,10 @@
 using Microsoft.Kiota.Abstractions.Serialization;
-using Microsoft.Kiota.Abstractions.Store;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System;
 namespace Microsoft.Graph.Models {
-    public class AgreementCollectionResponse : BaseCollectionPaginationCountResponse, IBackedModel, IParsable {
-        /// <summary>Stores model information.</summary>
-        public IBackingStore BackingStore { get; private set; }
```

Comment:
I didn't see a mention of removing BackingStore in the notes. Is this expected in this case?

Reply:
Thanks for this @MIchaelMainer,

The property should not be generated since it existed in the base class so having it removed/not present should be the expected case. 
https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/9ea1e7fbbbb79511e538401368fdc654d54e021d/src/Microsoft.Graph/Generated/Models/BaseCollectionPaginationCountResponse.cs#L15

However, it looks like Kiota has added the property to the `AgreementAcceptanceCollectionResponse` class as well. I'll hold this open for now to investigate.

Reply:
Resolved with #1844